### PR TITLE
[TEST] update map org

### PR DESCRIPTION
**Pull request by bumblebee89**

### CODE REVIEW

**Review by lukaszgryglicki — CHANGES_REQUESTED** [review]
please sort this...

**Comment by bumblebee89** [comment]
@lukaszgryglicki sorted

File: map_org_names.yaml [hunk 1]
```diff
@@ -604,11 +609,12 @@ mappings:
   - ['^[[:space:]]*4paradigm[[:space:]]第四范式[[:space:]]*$','Beijing Wusi Chuangxiang Technology Co., Ltd.']
   - ['^[[:space:]]*kuaishou[[:space:]]*$','BEIJING YUANDING TECHNOLOGY CO.,LTD']
   - ['^[[:space:]]*isure[[:space:]]cloud[[:space:]]*$','Beijing Yunyun Zhidu Technology Co., Ltd.']
-  - ['^[[:space:]]*&yet[[:space:]]*$','Beijing Zhiren Tianxia Technology Co., Ltd.']
+  - ['^[[:space:]]*//////[[:space:]]*$','Beijing Zhiren Tianxia Technology Co., Ltd.']
```
Comment (lukaszgryglicki) [inline map_org_names.yaml:612]: Is this `//////` intended? What is it supposed to catch?

**Review by lukaszgryglicki — APPROVED** [review]
/lgtm but please see one comment, if this is intended then go ahead and merge.

File: map_org_names.yaml (hunk 1 above)
Comment (bumblebee89) [reply map_org_names.yaml:612]: yes, we have an org with the exact name `//////` in DA

```
id   |name  |src|op|
-----|------|---|--|
32224|//////|   |  |
```

**Comment by lukaszgryglicki** [comment]
Have you tested that actual regexp if it works as expected? I mean escaping etc. You can do it via:
```
select '//////' regexp '^[[:space:]]*//////[[:space:]]*$';
+----------------------------------------------------+
| '//////' regexp '^[[:space:]]*//////[[:space:]]*$' |
+----------------------------------------------------+
|                                                  1 |
+----------------------------------------------------+
1 row in set (0.13 sec)
```
I see in this case that it will match, but actually, some very complex REGEXPs should be tested like this, because I'm only reviewing, not testing each one of them and I don't have MySQL regexp parser in my eye :P :D but I want to avoid some disaster like using a REGEXP that matches almost all company names and similar tragedies.

**Comment by bumblebee89** [comment]
@lukaszgryglicki yes, same query i used in testing to to come up with row above. This is the query i used

```
select * from organizations
where name regexp '^[[:space:]]*//////[[:space:]]*$'
```

**Comment by lukaszgryglicki** [comment]
Seems like we have one circular:
```
{"text":"Added organization 'Power Home Remodeling Group' (id=77293), Deleted organization 'Power Home Remodeling Group' (id=77293), Organizations: added:1 renamed:0 deleted:1 skipped:0, Enrollments: conflicts:0 archive conflicts:0 updated:0"}
```

**Pull request merged**